metricbeat/module/kubernetes/state_container/state_container.go

```diff
@@ -40,9 +40,10 @@ var (
 
 	// Mapping of state metrics
 	mapping = &p.MetricsMapping{
+		MetricSetName: "state_container",
```

Comment:
make "state_container", "kube_pod_container" and "kube_pod_info" to be string contstants to protect against typos.